testcases/cve/stack_clash.c

```diff
@@ -47,7 +47,7 @@
 #include "tst_test.h"
 #include "tst_safe_stdio.h"
 
-static unsigned long PAGE_SIZE;
+static unsigned long PAGE_SIZE_tst;
 static unsigned long PAGE_MASK;
```

Comment:
Can we please just rename the variable to use lover case, i.e. `page_size`? Otherwise the change looks good.